src/codewhisperer/activation.ts

```diff
@@ -266,10 +271,18 @@ export async function activate(context: ExtContext): Promise<void> {
                             await vscode.commands.executeCommand('aws.codeWhisperer.refresh')
                             await showSsoSignIn.execute()
                         } else if (resp === CodeWhispererConstants.accessTokenMigrationDoNotShowAgain) {
+                            vscode.window.showInformationMessage(
```

Comment:
Add await here?

Reply:
When I was testing the popup shows fine without the await here. That said, it also works with the addition of await (and if this will avoid any potential async issues I'm happy to include). Added in latest commit.